site/source/docs/compiling/Building-Projects.rst

```diff
@@ -73,6 +73,28 @@ WebAssembly.
   not the native system compiler. If *emcc* is not used, you may need to modify
   the configure or cmake scripts.
 
+.. _building-projects-with-cmake:
+
+Using CMake with Emscripten
+==================================
+
+If you have not already, first, you have to install and activate the SDK tag you wish to use, and after, you must add the environment variables in your terminal.
+
+.. code-block:: bash
+
+  ./emsdk install latest  # Download and install the latest SDK. You probably already did this!
+  ./emsdk activate latest # Make the "latest" SDK "active" for the current user
+  source ./emsdk_env.sh   # Activate PATH and other environment variables in the existing terminal
+  
+After activating the environment variables, you will have the `emcmake` utility available on the terminal. You always should use `emcmake` before any CMake invocation. It will pass to CMake the correct toolchain for building your projects.  
+
+.. code-block:: bash
+
+  mkdir build && cd build
+  emcmake cmake ..
+  make
+
+Do not use `emmake` before `make` with CMake generated makefiles.
```

Comment:
why not?

Reply:
as far as I can tell all needed environment variables are set either in the configured makefile or because `emsdk_env` so there's no need to. I could rewrite to say "There's no need to use `emmake` with Makefiles generated by `emcmake cmake`".

Reply:
ah, other thing, I am mentioning `make` because last time I tried (long ago) using the cmake `--build` did not work. I don't know if it works now, it may be useful for people on Windows (in case they don't have `make` but have `ninja`).

Reply:
Yes, I would go as far as to say it is a bad idea to do this.

The Makefiles that are generated by cmake and by autoconf (configure) are already fully self-contained they don't need to want any environment variables set.  

`emmake` should be used with non-auto-generated, or hand written makefiles only.

Reply:
> ah, other thing, I am mentioning `make` because last time I tried (long ago) using the cmake `--build` did not work. I don't know if it works now, it may be useful for people on Windows (in case they don't have `make` but have `ninja`).

`cmake --build`  should certainly work.    Perhaps you want to show and example with `cmake --build`  and another example with `-G Ninja` that calls ninja directly (or `-G Unix Makefiles` + `make`).   The nice thing about `cmake --build` is that it works whatever `-G` was passed in.

Reply:
Sounds good to say "there is no need to use emmake on a makefile from cmake".